internal/store/peering_connections.go

```diff
@@ -176,11 +187,13 @@ func (s *Store) AWSContainers(projectID string) ([]atlasv2.CloudProviderContaine
 func (s *Store) GCPContainers(projectID string) ([]atlasv2.CloudProviderContainer, error) {
 	switch s.service {
 	case config.CloudService:
-		result, _, err := s.clientv2.NetworkPeeringApi.ListPeeringContainerByCloudProvider(s.ctx, projectID).
-			PageNum(0).
-			ItemsPerPage(maxPerPage).
-			ProviderName("GCP").
-			Execute()
+		params := &atlasv2.ListPeeringContainerByCloudProviderApiParams{
+			GroupId:      projectID,
+			ProviderName: pointer.Get("GCP"),
+			ItemsPerPage: pointer.Get(maxPerPage),
+			PageNum:      pointer.Get(0),
```

Comment:
can you confirm if this command has pagination and of not add it?

Reply:
same, only projectID as param